go/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/cncf/udpa/go
```

Comment:
NB: go module declaration cannot be placed at the root since go does not like C++ files in test/